PINCE.py

```diff
@@ -3894,6 +3902,7 @@ def copy_to_clipboard(row, column):
         action = menu.exec(event.globalPos())
         actions = {
             switch_to_stacktrace: lambda: self.set_stack_widget(self.StackTrace),
+            toggle_stack_pointer: lambda: toggle_stack_from_sp_bp(),
```

Comment:
This doesn't work. Instead, only after I did the above comment, I had to use `toggle_stack_from_sp_bp(self)`  to make it work, `self.toggle_stack_from_sp_bp()` didn't work

Reply:
`self.toggle_stack_from_sp_bp()` should work. What error are you facing?

Reply:
```
Traceback (most recent call last):
  File "/home/random/PINCE/PINCE.py", line 3914, in tableWidget_Stack_context_menu_event
    actions[action]()
  File "/home/random/PINCE/PINCE.py", line 3905, in <lambda>
    toggle_stack_pointer: lambda: self.toggle_stack_from_sp_bp(),
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'MemoryViewWindowForm' object has no attribute 'toggle_stack_from_sp_bp'
```

This error is expected as `toggle_stack_from_sp_bp()` is defined in a function, not in a class. `self` keyword will not work here.

Reply:
That's because you have a coding mistake there, you're still using `self.toggle_stack_from_sp_bp()` when that's not an instance variable, it's a class variable, remove the `self.`.

Reply:
Sorry, GitHub UI confused me. I've expanded it all and looked closer, you have to move the `toggle_stack_from_sp_bp()` out of `tableWidget_Stack_context_menu_event(self, event)`.

This belongs next to `update_stack()`